samples/sample7/template.json

```diff
@@ -0,0 +1,57 @@
+{
+  "dimensions": [
+    300,
+    400
+  ],
+  "bubbleDimensions": [
+    25,
```

Comment:
Instead of keeping a separate pdf sample, lets add the pdf into sample1 itself. Along with jpg and png images together it should run fine